config.md

```diff
@@ -0,0 +1,157 @@
+# The js-ipfs config file
+
+The js-ipfs config file is a JSON document, which is located in a repository
+(by default located at ~/.jsipfs/config), where the repository location may be
+changed according to the value of the $IPFS_PATH environment variable.
+It is read once at node instantiation, either for an offline command, or when
+starting the daemon. Commands that execute on a running daemon do not read the
+config file at runtime.
+
+## Table of Contents
+
+- [`Addresses`](#addresses)
+- [`Bootstrap`](#bootstrap)
+- [`Datastore`](#datastore)
+- [`Discovery`](#discovery)
+- [`Identity`](#identity)
+- [`Swarm`](#swarm)
+
+
+## `Addresses`
+Contains information about various listener addresses to be used by this node.
+
+- `API`
+Multiaddr describing the address to serve the local HTTP API on.
```

Comment:
I suggest adding one sentence to each of the Addresses (API, Gateway, etc) explaining what it is. eg what is the HTTP API? (like you did for Bootstrap)